docs/community/ecosystem.md

```diff
@@ -0,0 +1,59 @@
+# MXNet's Ecosystem
+
+Community contributions to MXNet have added many new valuable features and functionality to support use cases such as model serving & portability, easy and flexible APIs, and educational material like crash courses and online books. This ecosystem page lists the projects that use MXNet, teach MXNet, or augment MXNet in some way.
+
+
+## Highlighted Project
+
+[![Promo image](https://cdn-images-1.medium.com/max/800/1*PwIMdZM7tpt3rmcyhlk2FQ.png)](https://medium.com/apache-mxnet/announcing-keras-mxnet-v2-2-4b8404568e75)
+#### [Keras-MXNet v2.2 released!](https://medium.com/apache-mxnet/announcing-keras-mxnet-v2-2-4b8404568e75)
+
+## Contents
+
+* [Learning MXNet and other Educational Resources](#learning-mxnet-and-other-educational-resources)
+* [New MXNet APIs](#new-mxnet-apis)
+* [Toolkits to Extend MXNet](#toolkits-to-extend-mxnet)
+* [Model Serving](#model-serving)
+* [Model Zoos](#model-zoos)
+* [MXNet as a Backend](#mxnet-as-a-backend)
+* [Contributions](#contributions)
+
+## Learning MXNet and other Educational Resources
+
+* [Gluon 60 Minute Crash Course](https://gluon-crash-course.mxnet.io/) - deep learning practitioners can learn Gluon quickly with these six 10-minute tutorials.
+* [The Straight Dope](https://gluon.mxnet.io/) - a series of notebooks designed to teach deep learning using the Gluon Python API for MXNet.
+
+
+## New MXNet APIs
+
+* [Clojure API](https://github.com/apache/incubator-mxnet/tree/master/contrib/clojure-package) - use MXNet with Clojure.
+* [Gluon Python Interface](../gluon/index.html) - train complex models imperatively and then deploy with a symbolic graph.
+* [ONNX-MXnet API](../api/python/contrib/onnx.html) - train and use Open Neural Network eXchange (ONNX) model files.
+* [Scala Infer API](../api/scala/infer.html) - model loading and inference functionality.
+
+
+## Toolkits to Extend MXNet
+
+* [Gluon CV](https://gluon-cv.mxnet.io/) - state-of-the-art deep learning algorithms in computer vision.
+* [Gluon NLP](https://gluon-nlp.mxnet.io/) - state-of-the-art deep learning models in natural language processing.
+
+
+## Model Serving
+
+* [MXNet Model Server (MMS)](https://github.com/apache/incubator-mxnet/tree/master/example/model-server/mms.md) - simple yet scalable solution for model inference.
+
+
+## Model Zoos
+
+* [Gluon Model Zoo](https://mxnet.incubator.apache.org/api/python/gluon/model_zoo.html) - models trained in Gluon and available through Gluon's model zoo API.
+* [ONNX Model Zoo](https://github.com/onnx/model-zoo) - ONNX models from a variety of ONNX-supported frameworks.
+
+
+## MXNet as a Backend
```

Comment:
Not sure this needs it's own section if we're already listing APIs above.

Reply:
You're referring to Keras-MXNet? I don't have it listed as an API. It seemed that MXNet might be used as a backend for other frameworks, so that's why I gave it its own section. Would you move it to APIs instead?

Reply:
When we get some more front-end frameworks using MXNet we can create section, but for now I would have in API section.